pandas/core/algorithms.py

```diff
@@ -1105,12 +1105,22 @@ def _get_score(at):
 def check_percentile(q: Union[float, Iterable[float]]) -> np.ndarray:
     """
     Validate percentiles (used by describe and quantile).
-    Args:
-        q: A single percentile or an iterable of percentiles.
-    Returns:
-        ndarray
-            An ndarray of the percentiles if valid.
-    Raises: ValueError if percentiles are not in given interval([0, 1]).
+
+    This function checks if the given float oriterable of floats is a valid percentile otherwise raises a ValueError.
+
+    Args
```

Comment:
Should be `Parameters` instead of `Args`

Reply:
Will update this asap